.circleci/publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
this file is copied and adapted from https://github.com/camel-tooling/camel-lsp-client-eclipse/blob/master/.travis_publish